common/api/ecschema-editing.api.md

```diff
@@ -95,7 +95,7 @@ export type AnySchemaDifference = SchemaDifference | SchemaReferenceDifference |
 export type AnySchemaEdits = SkipEdit | RenameSchemaItemEdit | RenamePropertyEdit;
 
 // @alpha
-export type AnySchemaItemDifference = AnyClassItemDifference | ConstantDifference | EnumerationDifference | EntityClassMixinDifference | FormatDifference | KindOfQuantityDifference | InvertedUnitDifference | PhenomenonDifference | PropertyCategoryDifference | UnitDifference | UnitSystemDifference;
+export type AnySchemaItemDifference = AnyClassItemDifference | ConstantDifference | EnumerationDifference | EntityClassMixinDifference | FormatDifference | KindOfQuantityDifference | KoqPresentationFormatDifference | InvertedUnitDifference | PhenomenonDifference | PropertyCategoryDifference | UnitDifference | UnitSystemDifference;
```

Comment:
KoqPresentationFormatDifference shouldn't be an AnySchemaItemDifference, rather put it in the AnySchemaDifference union like EntityClassMixinDifference which moved there with a recent fix too